exercises/generator_iterator/problem.fr.md

```diff
@@ -0,0 +1,45 @@
+Générateurs vs. Itérateurs.  Quelle est la différence ?
+
+## Infos
+
+Le principal truc à savoir, c'est que **les générateurs sont des itérateurs**.
+Ils se contentent d’enrober un objet itérateur (voir « Le protocole itérateur »,
```

Comment:
"Ils se contentent d’enrober un objet itérateur" -> "Ils sont une simple surcouche par dessus un objet itérateur"